ixmp/core.py

```diff
@@ -333,41 +339,63 @@ def add_timeseries(self, df, meta=False):
                 self._jobj.addTimeseries(df.region[i], df.variable[i], time,
                                          jData, df.unit[i], meta)
 
-    def timeseries(self, IAMC=False, node=None, variable=None):
-        """retrieve data from this TimeSeries instance as a pandas dataframe
+    def timeseries(self, iamc=False, regions=None, variables=None, units=None,
```

Comment:
Just some small consistency issues:
in the function iamc is in small letters while it is written in capital in the description below. In the function regions; in the description below regions.